### PR TITLE
Add edit reservation series all times

### PR DESCRIPTION
**Pull request by joonatank**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- Add: rescheduling a reservation series to `admin-ui`.
- Fix: enable type checking tests (ts lint).
- Fix: query single select filter type error when using `number` values.
- Fix: reservation title tag for a series not using the series start / end times.
- Remove unused utility code.
- Remove `SortedSelect` prefer sorting options when they are fetched (with backend if possible).

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of what's the fastest way to test your changes."

- Manual testing: both creation of new reservation series and editing an existing series `admin-ui`

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)"

- [TILA-3516](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3516)

[TILA-3516]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/admin-ui/src/spa/reservations/[id]/series/index.tsx [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+
+export const RescheduleReservationSeries = gql`
+  mutation RescheduleReservationSeries(
+    $input: ReservationSeriesRescheduleMutationInput!
+  ) {
+    rescheduleReservationSeries(input: $input) {
+      pk
+    }
+  }
+`;
```
Comment (joonatank) [inline apps/admin-ui/src/spa/reservations/[id]/series/index.tsx:100]: Could move to the end of file.

File: apps/admin-ui/src/spa/reservations/[id]/series/index.tsx [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+  endingDate: z.string(),
+  startTime: z.string(),
+  endTime: z.string(),
+  repeatOnDays: z.array(z.number()),
+  // TODO should have enable in the name
+  bufferTimeBefore: z.boolean(),
+  bufferTimeAfter: z.boolean(),
+  type: ReservationTypeSchema,
+  repeatPattern: z.literal("weekly").or(z.literal("biweekly")),
+});
```
Comment (joonatank) [inline apps/admin-ui/src/spa/reservations/[id]/series/index.tsx:116]: does the validation work though?

**Review by vincit-matu — APPROVED** [review]
🚀

File: apps/admin-ui/src/spa/reservations/[id]/series/index.tsx (hunk 2 above)
Comment (joonatank) [reply apps/admin-ui/src/spa/reservations/[id]/series/index.tsx]: fixed

**Pull request merged**